acstools/findsat_mrt.py

```diff
@@ -1360,6 +1360,12 @@ def rebin(self):
 
         LOG.info('Rebinning the data by {}'.format(self.binsize))
 
+        # trigger a warning if the image dimensions are not even
+        if ((self.image.shape[0] % self.binsize != 0)) | \
+           ((self.image.shape[1] % self.binsize != 0)):
+            LOG.warning('Warning: image dimensions do not evenly divide by bin'
+                        'size.\n Extra rows/columns will be trimmed first')
```

Comment:
No need to put "Warning:" because logger will make that clear.
```suggestion
            LOG.warning('Image dimensions do not evenly divide by bin '
                        'size.\nExtra rows/columns will be trimmed first.')
```